application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/Translations.xml

```diff
@@ -47,6 +47,9 @@ confluencepro.extension.notMainWiki.link.label=main wiki
 confluencepro.zippicker.title=Select a confluence package to start the migration
 confluencepro.zippicker.placeholder=Pick a confluence export package...
 confluencepro.zippicker.button.submit=Migrate
+confluencepro.zippicker.fileWarningEmpty=It looks like you did not select any archive to import!
+confluencepro.zippicker.fileWarningWrongFormat=It looks like you are trying to import a {0} file. The Confluence
```

Comment:
Note: the application name is "Confluence Migrator Pro", this message makes it look like the name is "Confluence Migrator Application" (probably because of the capitalization of "Application".